src/core/list/components/Title.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
```

Comment:
Components should be named `lower-case-component-with-kebab-case.jsx`

Reply:
Done.